api/DTOs/SupplierDTO.cs

```diff
@@ -0,0 +1,28 @@
+using Microsoft.AspNetCore.Mvc;
+
+namespace DTO.Supplier;
+
+[ApiExplorerSettings(IgnoreApi = true)]
+public class SupplierRequest : BaseDTO
+{
+    public string? Code { get; set; }
+    public string? Name { get; set; }
+    public string? Reference { get; set; }
+    public Guid? Contact_id { get; set; }
```

Comment:
```suggestion
    public Guid? ContactId{ get; set; }
```

Graag PascalCase aanhouden